docs/bin/testing_formatter.sh

```diff
@@ -1,12 +1,47 @@
 #!/bin/bash -eu
 
-cat <<- EOF > ../docsite/rst/dev_guide/testing/sanity/index.rst
+cat <<- 'EOF' > ../docsite/rst/dev_guide/testing_sanity.rst
```

Comment:
I think it makes more sense to move the testing_sanity contents to the sanity index.

@dharmabumstead Thoughts?

Reply:
I might have missed something but it seems `sanity/index.rst` wasn't referenced anywhere.

Reply:
I think it should be referenced instead of the old testing_sanity page.

Reply:
Where should be put the reference ?

Note that `dev_guide/testing/compile/index.rst` isn't referenced too and refers to `testing_compile` (this one is referenced).
```
Compile Tests
=============

See :doc:`../../testing_compile` for more information.
```

Reply:
@mattclay that works for me.